src/erlcloud_ec2.erl

```diff
@@ -2415,6 +2420,178 @@ request_spot_instances(Request, Config) ->
             {error, Reason}
     end.
 
+
+decode_spot_fleet_state("submitted") -> submitted;
+decode_spot_fleet_state("active") -> active;
+decode_spot_fleet_state("cancelled") -> cancelled;
+decode_spot_fleet_state("failed") -> failed;
+decode_spot_fleet_state("cancelled_running") -> cancelled_running;
+decode_spot_fleet_state("cancelled_terminating") -> cancelled_terminating;
+decode_spot_fleet_state("modifying") -> modifying.
+
+encode_termination_policy(default) -> "Default";
+encode_termination_policy(no_termination) -> "noTermination".
+
+encode_allocation_strategy(lowest_price) -> "lowestPrice";
+encode_allocation_strategy(diversified) -> "diversified".
+
+extract_describe_spot_fleet_request(Doc) ->
+    [
+     {instance_id, get_text("instanceId", Doc)},
+     {spot_instance_request_id, get_text("spotInstanceRequestId", Doc)},
+     {instance_type, get_text("instanceType", Doc)}
+    ].
+
+-spec(describe_spot_fleet_instances/1 :: (string()) -> {ok, [proplist()]} | {error, term()}).
+describe_spot_fleet_instances(SpotFleetRequestId) ->
+    describe_spot_fleet_instances(SpotFleetRequestId, default_config()).
+
+-spec(describe_spot_fleet_instances/2 :: (string(), string()) -> {ok, [proplist()]} | {error, term()}).
+describe_spot_fleet_instances(SpotFleetRequestId, NextToken) ->
+    describe_spot_fleet_instances(SpotFleetRequestId, NextToken, undefined, default_config()).
+
+-spec(describe_spot_fleet_instances/3 :: (string(), string(), pos_integer()) -> {ok, [proplist()]} | {error, term()}).
+describe_spot_fleet_instances(SpotFleetRequestId, NextToken, MaxResults) ->
+    describe_spot_fleet_instances(SpotFleetRequestId, NextToken, MaxResults, default_config()).
+
+-spec(describe_spot_fleet_instances/4 :: (string(), string(), pos_integer(), aws_config()) -> {ok, [proplist()]} | {error, term()}).
+describe_spot_fleet_instances(SpotFleetRequestId, NextToken, MaxResults, Config) ->
```

Comment:
it would be really beneficial if you add `describe_spot_fleet_instances_all` right away also which will handle the token pagination and return the full list. Examples can be found in other modules